cmd/bnsd/scenarios/main_test.go

```diff
@@ -159,7 +159,9 @@ func initGenesis(filename string, addr weave.Address) (*tm.GenesisDoc, error) {
 		},
 		"multisig": []interface{}{
 			dict{
-				"sigs":                 []weave.Address{addr},
+				"participants": []interface{}{
```

Comment:
:+1:

Reply:
But isn't there some other reference to multisig in cmd/bnsd?
I can't find it, but I thought there was something else Alex updated in AT tests.

Reply:
Those are acceptance tests :smile: 

This is the only one that was failing. I was not searching for more places where this could be used.